doc/dlt_for_developers.md

```diff
@@ -27,7 +27,7 @@ minimal code example. Detailed information about the API can be found later in
 this document.
 
 ```
-#include <dlt.h>
+#include <dlt/dlt.h>
```

Comment:
How about this: Since there wasn't a CMake Config file until now, we could allow people to make a switch with this; if they use pkg-config, they'll need to use <dlt.h>, but if they use the CMake Config file, they'll need to use <dlt/dlt.h>.

There is no compatibility concerns then, and users may chose which approach they prefer.